src/topology/compact_open.lean

```diff
@@ -333,6 +333,10 @@ begin
   apply_instance
 end
 
+@[simp]
+lemma uncurry_apply [locally_compact_space β] (f : C(α, C(β, γ))) (a : α) (b : β) :
+  f.uncurry ⟨a, b⟩ = f a b := rfl
```

Comment:
This lemma only simplifies `f.uncurry (a, b)` but it doesn't simplify `f.uncurry p`. Is it intentional?

Reply:
Maybe using @[simps] here is a better approach

Reply:
@[simps] complains:
```
Invalid `simps` attribute. Target eq is not a structure
```

Reply:
The `simps` would go on the def above, and you would delete this lemma.

Reply:
If you merge master, that will now be called `_apply` instead of `_to_fun`